content/60.methods.md

```diff
@@ -11,170 +28,245 @@ In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becom
-For example, beyond building just a PDF, Manubot also renders the manuscript in HTML and docx <!--To Do: And tex?-->
-The HTML manuscript format offers several advantages over a static PDF to harmonize available resources that we were able to apply to specific problems of COVID-19.
-The integration of scite has made the expansive number of references more manageable by visually representing whether their results are contested or whether they have been corrected or retracted.
-Cross-referencing cited preprints with their reviews in the appendix is another.
-Docx is a necessary format for a biological collaboration where authors are typically not working in LaTeX.
-<!--To Do: continue expanding to describe docx updates and possibly LaTeX updates-->
+The current project was managed through GitHub (https://github.com/greenelab/covid19-review) using Manubot [@doi:10.1371/journal.pcbi.1007128] to continuously generate a version of the manuscript online [@url:https://greenelab.github.io/covid19-review].
+The Manubot framework facilitated a massive collaborative review on an urgent topic.
+This project demonstrates that Manubot can be applied to projects where not all collaborators have expertise or even experience working with version control pipelines.
+Through the development of cyberinfrastructure both for training novice users to interact with GitHub and to simplify the workflows to allow them to receive many of the benefits of What You See Is What You Get platforms such as Google Docs, we were able to adapt a powerful open publishing tool to harness the domain expertise of a large group of non-technical users and to respond to the flood of COVID-19 publications.
 
-With the worldwide scientific community uniting during 2020 to investigate SARS-CoV-2 and COVID-19 from a wide range of perspectives, findings from many disciplines are relevant on a rapid timescale to a broad scientific audience.
+While Manubot manuscripts are written in markdown, they can be rendered in several formats that provide different advantages.
+For example, beyond building just a PDF, Manubot also renders the manuscript in HTML, docx, and now, LaTeX.
+The HTML manuscript format offers several advantages over a static PDF to harmonize available resources that we were able to apply to specific problems of COVID-19.
+The integration of scite into the HTML build makes references more manageable by visually representing whether their results are contested or whether they have been corrected or retracted.
+Cross-referencing different pieces of the manuscript, such as cited preprints with reviews stored in an appendix, is another dynamic option presented by HTML.
+Additionally, because of the heavy emphasis on Word processing in biology, Manubot's ability to generate docx outputs was expanded to allow users to generate docx files containing only a section of the manuscript.
```

Comment:
```suggestion
Additionally, because of the heavy emphasis on Word processing in biology, Manubot's ability to generate DOCX outputs was expanded to allow users to generate DOCX files containing only a section of the manuscript.
```